### ISSUE TITLE
fight enemies ( if if eligible & provoked )

### ISSUE THREAD

**Issue by baderror99** [MAFINS/MenyooSP#566]
fight enemies ( if if eligible & provoked ) 

object spooner in pad > task sequence > fight enemies ( if if eligible & provoked )

plz check this???

**Comment by baderror99** [comment]
this not working

**Comment by mistahwolf** [comment]
> fight enemies ( if if eligible & provoked )
> 
> object spooner in pad > task sequence > fight enemies ( if if eligible & provoked )
> 
> plz check this???

Yeah, that only works for the '3' main characters if you spawn them as a ped. Not sure why, but that's how it works.
